### DISCUSSION TITLE
This isn't chatGPT.

### DISCUSSION THREAD

**Discussion by kevin-huff** [jackMort/ChatGPT.nvim#27] [General]
text-davinci-003 is not chatGPT.

**Comment by ben-wall** [comment]
I think the point is that it will be once they release the API.

**Comment by jackMort** [comment]
Correct, I am currently using other available models, and when ChatGPT becomes available, I will switch to it. I have also been trying to use the unofficial API, but it has too many limitations.

**Reply by yngwi** [reply]
Will the other models stay available as well? I very much like being able to choose which one I want. They do have different advantages/disadvantages.
